data/presets/amenity/driver_training.json

```
@@ -0,0 +1,32 @@
+{
+    "icon": "maki-car",
+    "fields": [
+        "name",
+        "operator",
+        "operator/type",
+        "address",
+        "opening_hours"
+    ],
+    "moreFields": [
+        "license_classes",
+        "email",
+        "fax",
+        "opening_hours/covid19",
+        "payment_multi",
+        "phone",
+        "website",
+        "wheelchair"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "amenity": "driver_training"
+    },
+    "name": "Driver Training Grounds",
+    "aliases": [
+        "Driving Safety Center"
+        "Driving School"
```

hm, I would not add "Driving School" as an alias here, as this would collide with the preset for `amenity=driving_school` (note that [`aliases`](https://github.com/ideditor/schema-builder#aliases) are meant for synonyms).

maybe we could include "diving school ground" as a search [term](https://github.com/ideditor/schema-builder#terms)/keyword:

```suggestion
    ],
    "terms": [
        "Driving School"
```